keras_cv/models/backbones/resnet_v1/resnet_v1_backbone_presets_test.py

```diff
@@ -37,10 +37,6 @@ class ResNetPresetSmokeTest(tf.test.TestCase, parameterized.TestCase):
     def setUp(self):
         self.input_batch = tf.ones(shape=(2, 224, 224, 3))
 
-    @parameterized.named_parameters(
```

Comment:
It looks like after this change, there's no meaningful usage of `parameterized`, so we can probably remove it as a superclass of these tests and drop the import